src/ol/format/wfs/filter.js

```diff
@@ -0,0 +1,617 @@
+goog.provide('ol.format.wfs.filter');
```

Comment:
shouldn't we put this in ```ol.format.ogc.filter``` instead? Since filter can be used outside of wfs and is part of the ogc namespace not the wfs one.